client/src/App.js

```diff
@@ -89,15 +89,12 @@ class App extends Component {
         <div hidden="true" id="new-project-form">
           <form onSubmit={this.submitNewProject}> 
             <h2> Submit a new project</h2>
-            <label for="form-requester">Requester Name</label>
-            <input name="form-requester" id="form-requester"></input>
-            <label for="form-title">Title of Project</label>
-            <input name="form-title" id="form-title"></input>
-            <label for="form-content">tell us about the project</label>
+            <label for="form-requester">Requester name: <input name="form-requester" id="form-requester"></input></label>
+            <label for="form-title">Title of project: <input name="form-title" id="form-title"></input></label>
+            <label for="form-content">Tell us about the project</label>
             <textarea name="form-content" id="form-content"></textarea>
-            <label for="form-url">add a link to more information</label>
-            <input name="form-url" id="form-url"></input>
-            <input type="submit"/>
+            <label for="form-url">Add a link to more information: <input name="form-url" id="form-url"></input></label>
+            <input type="submit" value="Submit Request"/>
```

Comment:
I think we should give this the `button` styling, too...

Reply:
But it looks hard, so I filed https://github.com/FirefoxUX/photon-components-web/issues/9

Reply:
I'll just change it to a button